specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,392 @@
+          "type": "string",
+          "description": "Description for the incident."
+        },
+        "cloudInstanceId": {
+          "type": "integer",
+          "description": "CloudInstanceId or TenantSiloId for the incident 's tenant.",
+          "format": "int64"
+        },
+        "responsibleServiceId": {
+          "type": "integer",
+          "description": "Service responsible for the incident.",
+          "format": "int64"
+        },
+        "responsibleTeamId": {
+          "type": "integer",
+          "description": "Team responsible for the incident.",
+          "format": "int64"
+        },
+        "owningServiceId": {
+          "type": "integer",
+          "description": "Owning service identifier for the incident.",
+          "format": "int64"
+        },
+        "serviceCategoryId": {
+          "type": "integer",
+          "description": "Id of the service category of the service that owns this incident.",
+          "format": "int64"
+        },
+        "owningTeamId": {
+          "type": "integer",
+          "description": "Owning team identifier for the incident.",
+          "format": "int64"
+        },
+        "originatingServiceId": {
+          "type": "integer",
+          "description": "Originating service identifier for the incident.",
+          "format": "int64"
+        },
+        "isCustomerImpacting": {
+          "type": "boolean",
+          "description": "A value indicating whether this instance is customer impacting."
+        },
+        "isNoise": {
+          "type": "boolean",
+          "description": "A value indicating whether this instance is noise."
+        },
+        "isSecurityRisk": {
+          "type": "boolean",
+          "description": "A value indicating whether this instance is security risk."
+        },
+        "keywords": {
+          "type": "string",
+          "description": "Keywords for the incident."
+        },
+        "type": {
+          "type": "string",
+          "description": "Incident type."
+        },
+        "parentId": {
+          "type": "integer",
+          "description": "Parent incident id.",
+          "format": "int64"
+        },
+        "impactedServices": {
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "description": "A collection of impacted services."
+        },
+        "impactedTeams": {
+          "type": "array",
+          "items": {
+            "type": "integer",
+            "format": "int64"
+          },
+          "description": "A collection of impacted teams."
+        },
+        "incidentManagerContactId": {
+          "type": "integer",
+          "description": "Incident Manager' s contact id for this incident.",
+          "format": "int64"
+        },
+        "communicationsManagerContactId": {
+          "type": "integer",
+          "description": "Communication Manager 's contact id for this incident.",
```

Comment:
nit, you have Manager(space)'s then below you have Manager'(space)s

Reply:
addressed